docs/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Gebrüder Weiss Woocommere Plugin Documentation</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+  <meta name="description" content="Description">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
+  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/lib/themes/vue.css">
+</head>
+<body>
+  <div id="app"></div>
+  <script>
+    window.$docsify = {
+      name: 'Gebrüder Weiss Woocommerce Plugin Documentation',
+      logo: '/assets/images/logo.svg',
+      relativePath: true,
+      themeColor: '#f60',
+      repo: ''
+    }
+  </script>
+  <!-- Docsify v4 -->
+  <script src="//cdn.jsdelivr.net/npm/docsify@4"></script>
```

Comment:
I just have only one question: Why does docsify-cli have to be installed while it will be loaded through cdn at this line?

Reply:
@thai-phan-towa to serve it locally. It spins up a server on localhost:3000 where it serves from. Otherwise you have to manually link into the directory.

Reply:
I see, thanks for explaining!